Modules/Scripted/DICOMLib/DICOMWidgets.py

```diff
@@ -1141,10 +1132,10 @@ def configure(self):
     self.setColumnCount(3)
     self.setHorizontalHeaderLabels(['DICOM Data', 'Reader', 'Warnings'])
     self.setSelectionBehavior(qt.QTableView.SelectRows)
-    _setSectionResizeMode(self.horizontalHeader(), qt.QHeaderView.Stretch)
-    _setSectionResizeMode(self.horizontalHeader(), 0, qt.QHeaderView.ResizeToContents)
-    _setSectionResizeMode(self.horizontalHeader(), 1, qt.QHeaderView.Stretch)
-    _setSectionResizeMode(self.horizontalHeader(), 2, qt.QHeaderView.Stretch)
+    self.horizontalHeader(qt.QHeaderView.Stretch)
```

Comment:
While crafting the final PR, I noticed that this should be instead be:

```python
self.horizontalHeader().setSectionResizeMode(qt.QHeaderView.Stretch)
self.horizontalHeader().setSectionResizeMode(0, qt.QHeaderView.ResizeToContents)
self.horizontalHeader().setSectionResizeMode(1, qt.QHeaderView.Stretch)
self.horizontalHeader().setSectionResizeMode(2, qt.QHeaderView.Stretch)
```

Reply:
Ups, yes!

Reply:
Changing the wiki...